src/cdk/a11y/a11y-module.ts

```diff
@@ -9,27 +9,12 @@
 import {PlatformModule} from '@angular/cdk/platform';
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
-import {ARIA_DESCRIBER_PROVIDER, AriaDescriber} from './aria-describer/aria-describer';
-import {CdkMonitorFocus, FOCUS_MONITOR_PROVIDER} from './focus-monitor/focus-monitor';
-import {
-  CdkTrapFocus,
-  FocusTrapDeprecatedDirective,
-  FocusTrapFactory,
-} from './focus-trap/focus-trap';
-import {InteractivityChecker} from './interactivity-checker/interactivity-checker';
-import {LIVE_ANNOUNCER_PROVIDER} from './live-announcer/live-announcer';
+import {CdkMonitorFocus} from './focus-monitor/focus-monitor';
+import {CdkTrapFocus, FocusTrapDeprecatedDirective} from './focus-trap/focus-trap';
 
 @NgModule({
   imports: [CommonModule, PlatformModule],
   declarations: [CdkTrapFocus, FocusTrapDeprecatedDirective, CdkMonitorFocus],
   exports: [CdkTrapFocus, FocusTrapDeprecatedDirective, CdkMonitorFocus],
-  providers: [
-    InteractivityChecker,
-    FocusTrapFactory,
-    AriaDescriber,
-    LIVE_ANNOUNCER_PROVIDER,
-    ARIA_DESCRIBER_PROVIDER,
```

Comment:
I'm not super familiar with how the app-wide providers work yet, but should the `AriaDescriber` still be set in the `providers` in this case? I believe that it's the only reason why, for example, the tooltip imports the `A11yModule`.

Reply:
As I understand it, module imports just for this kind of injectable shouldn't be necessary any more. I'll address those in my next PR that applies this to material.